config.go

```diff
@@ -443,21 +442,15 @@ type moduleConfig struct {
 	// environKeys allow overwriting of existing values.
 	environKeys map[string]int
 
-	// preopenFD has the next FD number to use
-	preopenFD uint32
-	// preopens are keyed on file descriptor and only include the Path and FS fields.
-	preopens map[uint32]*wasm.FileEntry
-	// preopenPaths allow overwriting of existing paths.
-	preopenPaths map[string]uint32
+	*wasm.FSConfig
```

Comment:
Let's name fields instead of embedding. If this needs to be embedded, it would need a comment saying why.

```suggestion
	fs *wasm.FSConfig
```

Reply:
Done https://github.com/tetratelabs/wazero/pull/571/commits/588f6f8e65ab479cb92ac40f1e25fc3dbfc20b1a